setup.py

```diff
@@ -0,0 +1,47 @@
+#!/usr/bin/env python
+# -*- coding: utf-8 -*-
+try:
+    from setuptools import setup
+except ImportError:
+    from distutils.core import setup
+
+packages = [
```

Comment:
Can we either get rid of this single-use variable or turn it in to an `UPPER_CASE` constant?